app-header/app-header.html

```diff
@@ -116,9 +111,7 @@
       opacity: 0;
       box-shadow: inset 0px 5px 6px -3px rgba(0, 0, 0, 0.4);
       will-change: opacity;
-      -webkit-transition: opacity 0.4s;
       transition: opacity 0.4s;
-      -webkit-transform: translateZ(0);
```

Comment:
I tested the app-header demos on Safari 8 (my Yosemite VM) and the iOS 9 simulator - I didn't see any visual changes on them